app/main.py

```diff
@@ -27,12 +27,12 @@ def __iadd__(self, other: Distance | int | float) -> Distance:
     def __mul__(self, other: int | float) -> Distance:
         if isinstance(other, (int, float)):
             return Distance(self.km * other)
-        return Distance(self.km * other)
+        raise "Error"
 
     def __truediv__(self, other: int | float) -> Distance:
         if isinstance(other, (int, float)):
             return Distance(round(self.km / other, 2))
-        return Distance(round(self.km / other, 2))
+        raise "Error"
```

Comment:
You'll get error: "Exception should inherit from BaseException class" if it reaches raise statement(after raise there should be a class that inherits from BaseException class). Let's raise NotImplementedError